galaxy_ng/app/access_control/statements/insights.py

```diff
@@ -90,31 +90,36 @@
             "action": ["list"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": ["has_model_perms:galaxy.view_synclist", "has_rh_entitlements"]
+            "condition": ["has_model_perms:galaxy.view_synclist",
+                          "has_rh_entitlements",
+                          "is_org_admin"]
```

Comment:
any reason to have `has_rh_entitlements` in these statements when `is_org_admin` already calls underlying entitlement check?

Reply:
is_org_admin requiring has_rh_entitlements (essentially...) is kind of an implementation detail of is_org_admin. 
is_org_admin and has_rh_entitlement are slightly abstract so in theory, is_org_admin may not required has_rh_entitlement.

But mainly just to be consistent with the other access policies that require has_rh_entitlements as a separate and sometimes only condition. It's a little bit more explicit.